code/modules/atmospherics/machinery/components/fusion/hfr_procs.dm

```diff
@@ -15,6 +15,12 @@
 		if(. == FALSE)
 			break
 
+		if(syndicate_machine && !object.syndicate_machine)
+			. = FALSE
+
+		if(!syndicate_machine && object.syndicate_machine)
```

Comment:
what the fuck does this bit do

Reply:
so it wont fucking connect the normal hfr and syndi hfr together

Reply:
dont worry i tested all of them

Reply:
amazing, didnt even know that'd be needed